resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,56 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
```

Comment:
Using the new bootstrap 5 you only need to define this grid classes as “col-8” as default, in case you don’t need other screen sizes

Reply:
Cool. Thanks for the tip.